barman/postgres.py

```diff
@@ -893,25 +886,27 @@ def fetch_remote_status(self):
         try:
             # Retrieve wal_level, hot_standby and max_wal_senders
             # only if version is >= 9.0
-            if self.server_version >= 90000:
-                pg_settings.append("wal_level")
-                pg_settings.append("hot_standby")
-                pg_settings.append("max_wal_senders")
-                # Retrieve wal_keep_segments from version 9.0 onwards, until
-                # version 13.0, where it was renamed to wal_keep_size
-                if self.server_version < 130000:
-                    pg_settings.append("wal_keep_segments")
-                else:
-                    pg_settings.append("wal_keep_size")
-
-            if self.server_version >= 90300:
-                pg_settings.append("data_checksums")
-
-            if self.server_version >= 90400:
-                pg_settings.append("max_replication_slots")
-
-            if self.server_version >= 90500:
-                pg_settings.append("wal_compression")
+            pg_settings.extend(
+                [
+                    "wal_level",
+                    "hot_standby",
+                    "max_wal_senders",
+                ]
+            )
+            # Retrieve wal_keep_segments from version 9.0 onwards, until
+            # version 13.0, where it was renamed to wal_keep_size
+            if self.server_version < 130000:
+                pg_settings.append("wal_keep_segments")
+            else:
+                pg_settings.append("wal_keep_size")
+
+            pg_settings.extend(
```

Comment:
These elements could be added in the first `pg_settings.extend` call so that all new elements which are no longer version dependent are added in one place.

Reply:
I was not sure if order had meaning somehow and wanted to minimize the risk.  I will do that